libraries/Microsoft.Bot.Schema/Teams/MeetingDetails.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Note: The inheritance around these new classes/interfaces is based around how Teams sends different but similar payloads for `GetMeetingDetails` and the meeting start/end events.

See review comments (some resolved) for [the JS PR](https://github.com/microsoft/botbuilder-js/pull/3757) for previous discussion.